tensorflow/compiler/xla/service/cpu/compiler_functor.cc

```diff
@@ -22,6 +22,9 @@ limitations under the License.
 #include <utility>
 #include <vector>
 
+#include "external/llvm/tools/polly/include/polly/LinkAllPasses.h"
```

Comment:
These need to be conditional on whether polly is enabled right?

Reply:
This has not been addressed.

I've been looking for a way to define a macro like so - `ENABLE_POLLY` with bazel, in which case I could wrap this block of code with -

    #ifdef ENABLE_POLLY
    #include "external/llvm/tools/polly/include/polly/LinkAllPasses.h"
    .
    .
    #endif

Suggestions?

Reply:
Just like you have `append_polly_dep_if_enabled` you should be able to write a `POLLY_DEFINES_IF_ENABLED` which will expand to `"ENABLE_POLLY"` if polly is enabled (and empty string otherwise).  You can then use `defines` (https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library.defines) to pass this in to the `"compiler_functor"` build rule.